rosdep/base.yaml

```
@@ -4548,6 +4548,10 @@ qtmultimedia5-dev:
   freebsd: [qt5-multimedia]
   gentoo: ['dev-qt/qtmultimedia:5']
   ubuntu: [qtmultimedia5-dev]
+qttools5-dev-tools:
+  debian: [qttools5-dev-tools]
+  fedora: [qt5-qttools-devel]
```

qt5-qttools-devel is also part of RHEL 7:
```
$ yum list available -q --showduplicates qt5-qttools-devel
Available Packages
qt5-qttools-devel.x86_64                    5.9.2-1.el7                     base
```

```suggestion
  fedora: [qt5-qttools-devel]
  rhel: [qt5-qttools-devel]
```